wire-schema/src/test/java/com/squareup/wire/schema/ProtoTypeTest.kt

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (C) 2015 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire.schema
+
+import org.junit.Test
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Assert.fail
+
+class ProtoTypeTest {
+    @Test
+    @Throws(Exception::class)
+    fun get() {
+        assertThat(ProtoType.get("int32")).isSameAs(ProtoType.INT32)
+        assertThat(ProtoType.get("Person")).isEqualTo(ProtoType.get("Person"))
+        assertThat(ProtoType.get("squareup.protos.person", "Person"))
+                .isEqualTo(ProtoType.get("squareup.protos.person.Person"))
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun simpleName() {
+        val person = ProtoType.get("squareup.protos.person.Person")
+        assertThat(person.simpleName()).isEqualTo("Person")
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun scalarToString() {
+        assertThat(ProtoType.INT32.toString()).isEqualTo("int32")
+        assertThat(ProtoType.STRING.toString()).isEqualTo("string")
+        assertThat(ProtoType.BYTES.toString()).isEqualTo("bytes")
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun nestedType() {
+        assertThat(ProtoType.get("squareup.protos.person.Person").nestedType("PhoneType"))
+                .isEqualTo(ProtoType.get("squareup.protos.person.Person.PhoneType"))
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun primitivesCannotNest() {
+        try {
+            ProtoType.INT32.nestedType("PhoneType")
+            fail()
+        } catch (expected: UnsupportedOperationException) {
+        }
+
```

Comment:
nit: empty line. Same in a few other test methods.

Reply:
fixed